src/pages/[slug]/index.tsx

```diff
@@ -1,694 +1,83 @@
-	}, [query]);
-
-	const activeSort: SortObject = {
-		orderProp: query.orderProp,
-		orderDirection: (query.orderDirection as OrderDirection) ?? undefined,
-	};
-
 	/**
 	 * Data
 	 */
-
-	const {
-		error: visitRequestError,
-		data: visitRequest,
-		isLoading: visitRequestIsLoading,
-	} = useGetActiveVisitForUserAndSpace(slug as string, typeof slug === 'string');
-
-	const { data: visitorSpace, isLoading: visitorSpaceIsLoading } = useGetReadingRoom(
-		slug as string,
-		{ enabled: visitRequest !== undefined }
-	);
-
-	const { data: media } = useGetMediaObjects(
-		visitorSpace?.maintainerId?.toLocaleLowerCase() as string,
-		mapFiltersToElastic(query),
-		query.page || 0,
-		READING_ROOM_ITEM_COUNT,
-		activeSort,
-		visitorSpace?.maintainerId !== undefined
+	const { error: visitorSpaceError, isLoading: isVisitorSpaceLoading } = useGetVisitorSpace(
+		slug as string
 	);
 
-	const { data: filterOptions } = useGetMediaFilterOptions(
-		visitorSpace?.maintainerId?.toLocaleLowerCase() as string | undefined
-	);
-
-	// visit info
-	const { data: visitStatus } = useGetActiveVisitForUserAndSpace(router.query.slug as string);
-
-	const collections = useSelector(selectCollections);
-
 	/**
 	 * Computed
 	 */
 
-	const isNotFoundError = (visitRequestError as HTTPError)?.response?.status === 404;
-	const isNoAccessError = (visitRequestError as HTTPError)?.response?.status === 403;
-
-	/**
-	 * Effects
-	 */
-
-	useEffect(() => {
-		let buckets = filterOptions?.dcterms_format.buckets;
+	const isVisitorSpaceNotFoundError = (visitorSpaceError as HTTPError)?.response?.status === 404;
 
-		if (!buckets || buckets.length === 0) {
-			buckets = [
-				{ key: 'video', doc_count: 0 }, // Provide mock value for reduce
-			];
-		}
-
-		setMediaCount({
-			[ReadingRoomMediaType.All]: buckets
-				.map((pair) => pair.doc_count)
-				.reduce((p, c) => {
-					return p + c;
-				}),
-			[ReadingRoomMediaType.Audio]:
-				buckets.find((bucket) => bucket.key === ReadingRoomMediaType.Audio)?.doc_count || 0,
-			[ReadingRoomMediaType.Video]:
-				buckets.find((bucket) => bucket.key === ReadingRoomMediaType.Video)?.doc_count || 0,
-		});
-	}, [filterOptions]);
-
-	/**
-	 * Display
-	 */
-
-	const tabs: TabProps[] = useMemo(
-		() =>
-			READING_ROOM_TABS().map((tab) => ({
-				...tab,
-				label: (
-					<TabLabel
-						label={tab.label}
-						count={mediaCount[tab.id as ReadingRoomMediaType]}
-					/>
-				),
-				active: tab.id === query.format,
-			})),
-		[query.format, mediaCount]
-	);
-
-	const toggleOptions: ToggleOption[] = useMemo(
-		() =>
-			READING_ROOM_VIEW_TOGGLE_OPTIONS.map((option) => ({
-				...option,
-				active: option.id === viewMode,
-			})),
-		[viewMode]
-	);
+	// TODO make backend call to determine if content page exists and is accessible for the user
+	const isContentPageNotFoundError =
+		router.asPath === '/over-leeszalen' || router.asPath === '/faq';
+	const isContentPageLoading = false;
```

Comment:
Noteren in apart JIRA ticket om later op te nemen.

Reply:
https://meemoo.atlassian.net/browse/ARC-858